tests/service/test_service_without_skip_host_check.yml

```diff
@@ -0,0 +1,520 @@
+---
+- name: Test service without using option skip_host_check
+  hosts: ipaserver
+  become: yes
+
+  tasks:
+  # setup
+  - name: Get Domain from server name
+    set_fact:
+      ipaserver_domain: "{{ groups.ipaserver[0].split('.')[1:] | join ('.') }}"
+    when: ipaserver_domain is not defined
+
+  - name: Set host1, host2 and svc hosts fqdn
+    set_fact:
+      host1_fqdn: "{{ 'host1.' + ipaserver_domain }}"
+      host2_fqdn: "{{ 'host2.' + ipaserver_domain }}"
+      svc_fqdn: "{{ 'svc.' + ipaserver_domain }}"
+
+  - name: Host absent
+    ipahost:
+      ipaadmin_password: SomeADMINpassword
+      name:
+      # - svc.ihavenodns.info
+      - "{{ host1_fqdn }}"
+      - "{{ host2_fqdn }}"
+      - "{{ svc_fqdn }}"
+      update_dns: yes
+      state: absent
+
+  # If ansible-freeipa dnsrecord module is availble, it can be used.
+  - name: DNS record for "{{ svc_fqdn }}" is not present.
+    command: ipa dnsrecord-del --del-all "{{ ipaserver_domain }}" svc
+    ignore_errors: yes
+
+  - name: DNS record for "{{ host1_fqdn }}" is not present.
+    command: ipa dnsrecord-del --del-all "{{ ipaserver_domain }}" host1
+    ignore_errors: yes
```

Comment:
And this

Reply:
Fixed by removing usage of `ipa dnsrecord-*` and using ansible-freeipa host module.